src/wp-admin/about.php

```diff
@@ -219,7 +238,7 @@
 						printf(
 							/* translators: %s: WordPress version number. */
 							__( 'WordPress %s Release Notes' ),
-							'6.7'
+							$display_version
```

Comment:
This may need to stay `6.8` as the release notes URL in this section always points to the major version, rather than updating to the minor.

```suggestion
							'6.8'
```

Reply:
Ah, good catch, I forgot `$display_version` includes the minor.

> If you think it would be easier to set up a $display_major_version variable to avoid future changes

I've set this up to just be hardcoded for right now, at least changing it in one place is easier than finding it everywhere. This way also, it'll stay 6.8 even when trunk updates.